src/Symfony/Component/Validator/Resources/translations/validators.sv.xlf

```diff
@@ -366,6 +366,27 @@
                 <source>This value should be between {{ min }} and {{ max }}.</source>
                 <target>Detta värde bör ligga mellan {{ min }} och {{ max }}.</target>
             </trans-unit>
+            <trans-unit id="95">
+                <source>This value is not a valid hostname.</source>
+                <target>Värdet är inte ett giltigt server-namn.</target>
```

Comment:
```suggestion
                <target>Värdet är inte ett giltigt servernamn.</target>
```

Reply:
I was trying to find if hostname was translated before, but couldnt find any. "servernamn" is probably better.

Reply:
I know, it feels kind of awkward. Google translate says "värdnamn", but I would not really understand that. 

I would have used the English word "hostname", but that is also wrong. I think "servernamn" is a fair compromise.